static/styles/compose.css

```diff
@@ -262,6 +262,7 @@
 
     button {
         background: transparent;
+        color: inherit;
```

Comment:
You missed the `opacity` of `0.6`, which differs from the `0.7` for the other buttons; changed.